frontend/src/__tests__/cypress/cypress/pages/pipelines/topology.ts

```diff
@@ -8,28 +8,34 @@ import type {
 import { SearchSelector } from '~/__tests__/cypress/cypress/pages/components/subComponents/SearchSelector';
 
 class TaskDrawer extends Contextual<HTMLElement> {
+  private findTaskDrawer() {
+    return cy.findByTestId('task-drawer');
+  }
```

Comment:
Should chain from `this.find()` in `Contextual` classes.

```suggestion
  private findTaskDrawer() {
    return this.find().findByTestId('task-drawer');
  }
```